tekton-catalog/pipeline-loops/config/203-object-store-config.yaml

```diff
@@ -0,0 +1,29 @@
+# Copyright 2020 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: object-store-config
+  namespace: tekton-pipelines
+  labels:
+    app.kubernetes.io/instance: default
+    app.kubernetes.io/part-of: tekton-pipelines-loops
+data:
+  enable: "false"
+  apiKey: "<APIKEY-dummy-1231231231-123abcdefgh>"
```

Comment:
Can we also have an option to push to object storage using HMAC S3 keys. (e.g. `access_key_id` and `secret_access_key`?) So we can use this option with minio storage for open source as well.